.gitignore

```diff
@@ -9,3 +9,4 @@ langtest
 /pkg/languages/builders
 dry-run.json
 info.json
+test/info.json
```

Comment:
💡 Wondering/Question please: Why are we adding something in `gitignore` if we are removing the file? with validity commit, all users using this repo, will get this change on git pull. Thanks heaps.

Reply:
this file is created during a local run of the integration tests, so we need to exclude and rm

Reply:
☕️ Cool, thanks. Sounds good in that case rather than static file name let’s add a pattern? Like **.json because that will rule out any mistake, or is it just 1 file which is info.json.

https://git-scm.com/docs/gitignore#_pattern_format

Reply:
that's a great point, let's just keep the testing files in a dir and gitignore the whole dir instead